examples/prostate/prostate_3D/job_configs/prostate_fedprox/app/config/config_fed_server.json

```diff
@@ -15,7 +15,7 @@
       "model": {
             "path": "monai.networks.nets.unet.UNet",
             "args": {
-              "dimensions": 3,
+              "spatial_dims": 3,
```

Comment:
are we changing this because MONAI updates?

Reply:
yes, this results in a warning